### DISCUSSION TITLE
Open buffer in side window?

### DISCUSSION THREAD

**Discussion by strnglp** [minad/vertico#404] [Q&A] [answered]
I'm pretty new to emacs so it's likely I'm just doing this wrong. It seems like setting consult-grep to "buffer" does open it in a buffer, but it doesn't appear in the buffer list so I'm guessing it's somehow special and the display-buffer-alist cannot affect it. Can someone point me in the right direction here? Thanks.

```
(use-package vertico
    :bind (:map vertico-map
                ("C-j" . vertico-next)
                ("C-k" . vertico-previous))
    :config
    (setq vertico-resize nil
          vertico-multiform-categories
            '((consult-grep buffer)))
    :init
    (vertico-multiform-mode)
    (vertico-mode))

  ;; left, top, right, bottom
  (setq window-sides-slots '(0 0 1 0))

  (add-to-list 'display-buffer-alist
               '("^\\*[Gg]rep"
                 (display-buffer-in-side-window)
                 (side . right)
                 (slot . 0)
                 (window-parameters . ((no-delete-other-windows . t)))
                 (window-width . 0.5)))

```

**Comment by minad** [comment, marked as answer]
See vertico-buffer-display-action.

**Reply by eval-exec** [reply]
Hello, I didn't found `vertico-buffer-display-action`, is this function renamed?
I want to display vertico in side-window, but I don't know how to do.
